Example/SignUp/SignUpPresetner.swift

```diff
@@ -141,3 +152,15 @@ final class SignUpPresenter {
         }
     }
 }
+
+/// Async gender picker state.
```

Comment:
Not sure how this comment is relevant as well. The enum by itself has no relationship with where it is used.

Reply:
Same here https://github.com/Babylonpartners/Bento/pull/182#discussion_r292845574